packages/core/src/App/Containers/Layout/app-contents.jsx

```diff
@@ -112,6 +113,7 @@ const AppContents = observer(({ children }) => {
             })}
             ref={scroll_ref}
         >
+            {should_show_global_loader && <Loading is_fullscreen />}
```

Comment:
could you help me to know what we are fixing by adding all these loaders in different places @sergei-deriv 🙏

Reply:
@mitra-deriv tried to answer above